xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -39,9 +39,19 @@
         <Parameter Name="allowedCipherSuites" Type="System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;" />
       </Parameters>
       <Docs>
-        <param name="allowedCipherSuites">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="allowedCipherSuites">The collection of cipher suites allowed in this policy for negotiation.</param>
+        <summary>Initializes a cipher suite policy that can be used on a client or server authentication.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+> [!NOTE]
+> Defining a chipher suite policy on <xref:System.Net.Security.SslStream> authentication will prevent the OS of the ability to decide which are the best cipher suites to negotiate with and will need of your attention to manually check and update this code. It is strongly recommended to hold off the use of this property and rely on your constantly updated OS policy.
+
+## Remarks
+Unlike the OS underlying SSL/TLS implementation, the order of the <xref:System.Net.Security.TlsCipherSuite> elements in `allowedCipherSuites` does not determine their priority in the client-server negotiation. It is required to use OSX or Unix-based systems with OpenSSL 1.1.1 in order to initialize an instance of this class.
```

Comment:
I think the "or" here is lexically ambiguous.  IIRC it requires `((macOS) or (Linux with OpenSSL 1.1.1))`, but it's possible to parse as `((macOS or Linux) with OpenSSL 1.1.1)`.

It probably needs some sort of style bubble for OS-specific requirements.  Something like

```
[!IMPORTANT]
This feature is dependent on platform support.
* Microsoft Windows: This feature is not available.
* macOS: This feature is available.
* Linux: This feature is available on systems with OpenSSL 1.1.1 or higher.
```

But I'd wait for a writer before changing the format.

Reply:
I left a suggestion above. See what you think. I've changed from Unix-based to Linux since macOS it's also Unix-based.